_episodes/01-version-control.md

```diff
@@ -12,35 +12,88 @@ objectives:
 - "Familiarize with GitHub"
 - "Learn how to create a GitHub repository"
 keypoints:
-- "Version control systems help to organize and collaborate on code"
-- "Using a public repository contributes to the reproducibility, reusability and quality of your code"
+- "Version control systems help to organize and collaborate on code."
+- "Using a public repository contributes to the reproducibility, reusability and quality of your code."
 ---
-The [first principle](https://fair-software.eu/recommendations/repository) states:
-> ## Principle 1
-> Use a publicly accessible repository with version control
+
+> ## [Principle 1](https://fair-software.eu/recommendations/repository)
+>
+> Use a publicly accessible repository with version control.
 {: .callout}
+![principle1]({{ page.root }}/fig/principle1.png)
```

Comment:
change this to use 'recommendation' instead of 'principle' not to confuse with original FAIR data principles

Reply:
Done!